components/reddit/sources/new-comments-on-a-subreddit/new-comments-on-a-subreddit.js

```diff
@@ -0,0 +1,107 @@
+const common = require("../../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-on-a-subreddit",
+  name: "New comments on a subreddit",
+  description:
+    "Emits an event each time a new comment is added to a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    subreddit: {
+      propDefinition: [reddit, "subreddit"],
+    },
+    subreddit_id36: {
+      type: "string",
+      label: "ID36 of Subreddit",
+      description:
+        "Pipedream will emit a new event when a comment is posted to the subreddit pointed by this ID36.",
+      optional: false,
+    },
```

Comment:
Could this be renamed to `subredditArticle` or `subredditPost` instead? Those should be familiar enough to any user without them needing to inspect the Reddit API to find the meaning of ID36 or other details.

Also, we should accompany this with a helpful description, something like:
```
The ID of the post to monitor, which is present in the post URL as **r/<subreddit>/comments/<post ID>/...**
```